### ISSUE TITLE
MP4ThumbsAndGif Not working

### ISSUE THREAD

**Issue by para2022** [WWBN/AVideo#8721]
It is showing no thumbnails. I have updated it. Same problem. Only works if using Get jpeg/gif button. It is not doing it automatically anymore.

**Comment by DanielnetoDotCom** [comment]
do you mean when you directly upload an MP4?

**Comment by para2022** [comment]
Yes mate if a user uploads a direct upload then the thumbnail is blank, its a grey thumbnail. It doesnt generate a thumbnail now but it will generate them if i use the get jpeg/gif button. This doesnt affect the encoder though, just direct upload.

**Comment by DanielnetoDotCom** [comment]
I just tested on my side and MP4ThumbsAndGif works after uploading a video. 

I am not sure what happens on your side, but the MP4ThumbsAndGif version 4.3 I added some debugs, you can try to update the plugin, upload a new video and check the logs

**Comment by para2022** [comment]
Ok i will do that. Thank you
